plugins/devfile-plugin/src/main/kotlin/che/incubator/devfile/DevfileConfigurable.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2022 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package che.incubator.devfile
+
+import com.intellij.openapi.options.Configurable
+import com.intellij.openapi.ui.LabeledComponent
+import com.intellij.openapi.ui.VerticalFlowLayout
+import com.intellij.ui.components.JBTextField
+import java.awt.BorderLayout
+import java.awt.Dimension
+import javax.swing.JComponent
+import javax.swing.JPanel
+
+class DevfileConfigurable : Configurable {
+
+    private val settingsPanel = JPanel()
+
+    private val flattenedDevfileEnvVariableTextField = LabeledComponent.create(
+        JBTextField(),
+        "Flattened Devfile environment variable"
+    )
+
```

Comment:
The preferences page for editing `DEVWORKSPACE_FLATTENED_DEVFILE` environment variable name makes no sense.
I can't imagine a situation when the user would need to change it. This name is provisioned by the DevWorkapce Operator, AFAIK.

Reply:
Removed preference page.